src/features/claims/claims-list/ClaimListFilters.tsx

```diff
@@ -33,6 +41,33 @@ const FilterElement = styled.div`
   }
 `
 
+const StyledLabel = styled(Label)`
+  display: flex;
+  align-items: center;
+
+  & span {
+    margin-right: 1em;
+  }
+
+  & svg {
+    width: 15px;
+    height: 15px;
+
+    &:hover {
+      cursor: help;
+    }
+  }
+`
+
+const LabelWithPropower = ({ label, propover }) => (
+  <StyledLabel>
+    <span>{label}</span>
+    <Popover style={{ width: 250 }} contents={propover}>
+      <InfoCircle />
+    </Popover>
+  </StyledLabel>
```

Comment:
What if we make the entire label + icon show the Popover? Instead of having to aim with your mouse exactly on the icon, you can just hover on whatever. 

Also,

```suggestion
  <StyledLabel>
    <span>{label}</span>
    <Popover style={{ width: 250 }} contents={popover}>
      <InfoCircle />
    </Popover>
  </StyledLabel>
```

Reply:
@maxplt What do you think about this?

Reply:
@rasmusguterstam I tried, but I think it looks not so good. Don't know 🤷‍♂️
![image](https://user-images.githubusercontent.com/89198006/140055292-7bdce7cd-5737-4bbb-a1e3-f0634fd45961.png)

Reply:
Alright skip it then 😄